docs/reference/configuration.mdx

```diff
@@ -297,6 +304,78 @@ rate limiter. This could be combined with a second instance of
 
 </Note>
 
+## `ipv6Subnet`
+
+> `number` (32-64) | `function` | `false`
+
+IPv6 subnet mask applied to IPv6 addresses in the default
+[`keyGenerator`](#keygenerator).
+
+Generally, ISPs that support IPv6 give each of their customers a range of IPv6
+addresses, whereas they usually provide only a single IPv4 address per customer.
+A malicious user could iterate through their range of IPv6 addresses and bypass
+simple IP-based rate limiting. This setting counteracts that by allowing the
+rate-limiter to block an entire range of IPv6 addresses at once.
+
+The default value is 56, corresponding to a /56 subnet.
+
+The valid range is 1-128, but typically ISPs assign subnets in the range of
+32-64, with 64 being the most common. The validation check (which can be
+disabled) will warn for values outside the 32-64 range.
+
+In a 128 bit IPv6 address, the number of bits in this setting will be kept, and
+the rest will be ignored when determining the user's IP address. A smaller
+number means the ISP is reserving fewer bits for their control and allowing a
+larger portion of the address to be in the customer's control, thereby allowing
+the customer to have a greater number of unique IPs.
```

Comment:
I feel like this paragraph in particular is kind of confusing. I might try rewording this section a bit.